blog.md

```diff
@@ -1,26 +1,78 @@
+
+Actions speak louder than words so I decided to document (and demonstration) a Platform built from the ground up (using Open Source projects) and then deploying an application to it via a Continuous Integration/Delivery (CI/CD) pipeline.
+
+You could (and probably should) be using a public cloud provider for some and allof this stack, however I wanted to demonstrate that a system like this can be built and run internally as not everybody is able (for many valid reasons) able to use public cloud.
+
+As I wrote this I discovered that while figuring out the right combination of tools to run was a fun process, the really interesting stuff was building the actual CI/CD pipeline to deploy and run the application itself. This means that while I'll briefly describe the underlying infrastructure I will not be providing a detailed installation guide.
 
 ## Infrastructure
 
-1. [OpenStack](http://www.openstack.org/) was installed across a set of physical machines using [Ursula](https://github.com/blueboxgroup/ursula).
-2. A 3 node [DEIS](http://deis.io/)
- cluster was then installed on [OpenStack](http://www.openstack.org/) using the terraform instructions [here](https://github.com/paulczar/deis/tree/openstack_provision_script/contrib/openstack).
-3. A standalone coreos node was created using [Terraform](https://terraform.io/) 
-4. A self replicating database was deployed on the [DEIS](http://deis.io/) nodes using the [paulczar/percona-galera](https://github.com/paulczar/docker-percona_galera) docker image which uses [etcd](https://coreos.com/etcd/) to find each other and form a cluster.
-5. A mysql proxy was deployed on the standalone node using the [paulczar/maxscale](https://hub.docker.com/r/paulczar/maxscale/) image which discovers the databases via [etcd](https://coreos.com/etcd/).
-6. [Jenkins](http://jenkins-ci.org/) was deployed to standalone node using the standard [jenkins docker image](https://hub.docker.com/_/jenkins/).
+While an IaaS is not strictly necessary here (I could run Deis straight on bare metal) it makes sense to use something like [OpenStack](http://www.openstack.org/) as provides excellent APIs to spin up VMs.  I installed OpenStack across across a set of physical machines using [Blue Box's](https://www.blueboxcloud.com/) [Ursula](https://github.com/blueboxgroup/ursula).
+
+Next the PaaS itself, I have familiarity with [Deis](http://Deis.io/) already and I really like its (Heroku-esque) user experience.  I span up a three node Deis cluster on OpenStack using the terraform instructions [here](https://github.com/paulczar/Deis/tree/openstack_provision_script/contrib/openstack).
```

Comment:
« Next **is** the PaaS itself**.** I have [...] user experience. I sp**u**n up a [...] using the **T**erraform instructions [...] »
* Broke into two sentences.
* Fixed typo.
* Added capitalisation for a name.